packages/@guidesmiths/cuckoojs-schematics/README.md

```diff
@@ -26,3 +26,48 @@ npm publish
 ```
 
 That's it!
+
+## Available schematics
+
+### Pull Request Template
+
+* **Description**: Enables having a standard format when doing Pull Request
+* **Command**: `schematics @guidesmiths/cuckoojs-schematics:pr-template`
+* **Parameters**:
+
+| Name        | Description                       | Supported values        | Default value  |
+|-------------|-----------------------------------|-------------------------|----------------|
+| `gitProvider` | Git provider where repo is hosted | `github`, `azuredevops` | |
+
+
+### Dockerfile
+
+* **Description**: Created a Dockerfile for your build type
+* **Command**: `schematics @guidesmiths/cuckoojs-schematics:dockerfile`
+* **Parameters**:
+
+| Name          | Description                           | Supported values        | Default value | 
+|---------------|---------------------------------------|-------------------------|---------------|
+| `directory`   | Directory where files will be created | *valid folder*          | `.` |
+| `buildType`   | Specify the build process             | `github`, `azuredevops` | |
+| `nodeVersion` | Specify the NodeJS version to use     | `18`, `16`, `14`, `12`  | |
```

Comment:
Maybe we could use the latest LTS version as default

Reply:
I tried to set a default value when using a list type value but I couldn't, how could I do that?